src/components/layout/Stack/Stack.styled.tsx

```diff
@@ -22,39 +22,81 @@ export const Element = styled(
     forwardRef<HTMLDivElement, StackProps>(StyledStack)
 )`
     width: 100%;
-    flex-direction: ${({ flexDirection }) =>
-        flexDirection ? flexDirection : undefined};
 
-    ${({ justifyContent }) =>
-        justifyContent &&
+    ${({ flexDirection, direction, justifyContent, alignItems }) =>
+        (direction === 'horizontal' ||
+            flexDirection ||
+            justifyContent ||
+            alignItems) &&
         css`
             display: flex;
             justify-content: ${justifyContent};
+            align-items: ${alignItems};
+            flex-direction: ${direction === 'horizontal'
+                ? 'row'
+                : direction === 'vertical'
+                ? 'column'
+                : flexDirection};
         `}
 
-    ${({ alignItems }) =>
-        alignItems &&
+    ${({ breakpoints }) =>
+        breakpoints &&
         css`
-            display: flex;
-            align-items: ${alignItems};
+            ${Object.entries(breakpoints)
+                .map(
+                    ([breakpoint, { flexDirection, direction }]) =>
+                        (direction === 'horizontal' || flexDirection) &&
+                        `
+                            @media ${from[breakpoint]} {
+                                display: flex;
+                                flex-direction: ${
+                                    direction === 'horizontal'
```

Comment:
Müsste hier nicht auch noch der andere Case berücksichtigt werden, direction = 'vertical'? Evtl macht es Sinn das in eine Helper-Funktion auszulagern (wie orientation) - jedoch besser ausserhalb der StyledComponent. z.B. getFlexDirection